Fixes #688 Fixed create event button

geekychaser opened the pull request:
Fixes #688 Fixed create event button

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
Please Review
Thanks

niranjan94 commented on app/templates/components/nav-bar.hbs:
```diff
@@ -46,7 +46,7 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
```
Instead of this, make the whole item a link and add the `ui blue` class here so that it is styled correctly.

geekychaser replied:
sir i have not made any changes to package-lock.json but its showing 901 deletion. please help me.
thank you

niranjan94 replied:
Sometimes, when you run `npm install` it might change the file. Just revert the changes to that file in git.

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekychaser commented:
sir, i have reverted the package-lock.json file.
i have added the ui blue class to div itself hoping that this time it is styled correctly.
please review sir.
thank you.

niranjan94 commented on app/templates/components/nav-bar.hbs:
```diff
@@ -45,8 +45,8 @@
       {{#if (not-eq routing.currentRouteName 'create')}}
-        <div class="item">
-          {{#link-to 'create' class='ui blue button'}}
+        <div class="item ui blue">
```
Instead of a link-to inside a div, just have a link-to with the class `item ui blue`

geekychaser replied:
![image](https://user-images.githubusercontent.com/24489162/29749259-1632cbe6-8b45-11e7-9415-cefd49143648.png)
sir making whole class link-to is not making 'create event' blue in color due to css property.

geekychaser replied:
Sir the changes i earlier made using div tag performs and looks sames as this link: https://pr-786-open-event-frontend-fossasia.surge.sh

niranjan94 replied:
`ui blue` class does not exist. In this, it will look blue even without the class. Because blue is the default color of an anchor tag in semantic-ui.

But the issue here is, you're not able to click on any other place other than the text. Because you have put an anchor tag with a `div.item`. 

So, please don't have the `link-to` inside a div instead move the link-to outside of div add a class `ui blue` and style it blue via css.

geekychaser commented:
sir please check it.

On Sat, Aug 26, 2017 at 10:59 AM, Niranjan Rajendran <
notifications@github.com> wrote:

> *@niranjan94* requested changes on this pull request.
> ------------------------------
>
> In app/templates/components/nav-bar.hbs
> <https://github.com/fossasia/open-event-frontend/pull/786#discussion_r135382149>
> :
>
> > @@ -45,8 +45,8 @@
>          <a href="{{href-to 'login'}}" class="item">{{t 'Login'}}</a>
>        {{/if}}
>        {{#if (not-eq routing.currentRouteName 'create')}}
> -        <div class="item">
> -          {{#link-to 'create' class='ui blue button'}}
> +        <div class="item ui blue">
>
> Instead of a link-to inside a div, just have a link-to with the class item
> ui blue
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossasia/open-event-frontend/pull/786#pullrequestreview-58794644>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXWsyrwsJz-te1MOXs27Afv54KnvySL3ks5sb61KgaJpZM4PAKA7>
> .
>

geekychaser commented:
[image: Inline image 1]

sir i am not able to pass this test case. Even if my changes are working
perfectly and code is same as expected.

thank you sir

On Wed, Aug 30, 2017 at 12:57 AM, open-event-bot[bot] <
notifications@github.com> wrote:

> Hi @geekychaser <https://github.com/geekychaser>!
>
> It looks like one or more of your builds have failed.
> I've added the relevant info below to save you some time.
>
>    - Build 2934.1
>    <https://travis-ci.org/fossasia/open-event-frontend/jobs/269727120>
>    has failed
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossasia/open-event-frontend/pull/786#issuecomment-325772050>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXWsyjK_-pELZ0Qu9bfakZz9BhKtfwt2ks5sdGY9gaJpZM4PAKA7>
> .
>

niranjan94 commented on app/styles/partials/nav-bar.scss:
```diff
@@ -14,3 +15,7 @@
   border-radius: 0 !important;
 }
+.blue_text{
+ color:blue !important;
```
Use the correct semantic ui blue color.

niranjan94 replied:
Also keep the class name semantic. Name it `ui blue text`

niranjan94 commented on app/styles/partials/nav-bar.scss:
```diff
@@ -1,6 +1,7 @@
-.nav-bar > .item > .ui.blue.button {
+.nav-bar > .item > .ui.button {
```
This should point to that specific item

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekychaser commented:
sir , is it now working properly or some changes are being required?


On Fri, Sep 1, 2017 at 1:41 AM, Niranjan Rajendran <notifications@github.com
> wrote:

> Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossasia/open-event-frontend/pull/786#issuecomment-326407370>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXWsyqPH-44-QJAecpDRrbBcPXkT-hQHks5sdxNhgaJpZM4PAKA7>
> .
>

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekyd commented on app/styles/partials/nav-bar.scss:
```diff
@@ -14,3 +14,7 @@
 }
+
+.blue.text {
+  color: #425ff4 !important;
```
@geekychaser Please use `#2185D0` here

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekyd commented on app/templates/components/nav-bar.hbs:
```diff
@@ -19,10 +19,8 @@
-               alt="{{authManager.currentUser.email}}"
-               src="{{if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
-                         '/images/placeholders/avatar.png'}}">
+          {{widgets/safe-image class='ui avatar round image nav-bar-image' src=(if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
```
@geekychaser This change has already been made, seems like your branch is out of date. Can you manually rebase the branch?

geekyd replied:
The changes related to this were made in https://github.com/fossasia/open-event-frontend/pull/814

geekychaser replied:
sir rebased the branch and commited. Please review it.

thank you

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekychaser commented:
sir ,
please review it and suggest me if some more changes are required.

geekyd commented:
@geekychaser Just squash your commits. It looks good.

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

geekychaser commented:
sir,
all the commits are being squashed. please have a look.
thank you

niranjan94 commented:
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh
